src/egl/drivers/dri2/platform_android_gralloc.cpp

```diff
@@ -0,0 +1,33 @@
+#include "egl_dri2.h"
+
+extern "C" int update_pfn(struct dri2_egl_display *dri2_dpy);
+int update_pfn(struct dri2_egl_display *dri2_dpy){
+   const char *err;
+   int ret;
+   //const hw_module_t *gralloc1 ;
+   hw_device_t *device;
+
+   ret = hw_get_module(GRALLOC_HARDWARE_MODULE_ID,
```

Comment:
I think this and the following version check should be happening before this function is called, and it would be only called for gralloc1 case